app/qml/components/MMPhoto.qml

```diff
@@ -38,19 +38,21 @@ Image {
     }
   }
 
-  MouseArea {
+  Rectangle {
     anchors.fill: parent
-    onClicked: control.clicked(control.photoUrl)
+    color: __style.whiteColor
+    z: -1
```

Comment:
What is the reason to use `z`? I highly recommend not using that

Reply:
I think, using z here in the base component is safe. This just adds a white background with the 'waiting' icon when the picture is not loaded yet.